### ISSUE TITLE
node\r: No such file or directory

### ISSUE THREAD

**Issue by adelarsq** [rcjsuen/dockerfile-language-server-nodejs#198]
Trying to initiate the server in the console on OSX shows an error:

```
→ docker-langserver --stdio
env: node\r: No such file or directory
```
Related issue https://github.com/nwjs-community/nw-builder/issues/75

**Update:**
Solved with:

```
→ cd  ~/.config/yarn/global/node_modules/dockerfile-language-server-nodejs/bin
→ sudo dos2unix docker-langserver
dos2unix: converting file docker-langserver to Unix format...
```

**Comment by rcjsuen** [comment]
Thank you for the bug report, @adelarsq. Looks like the file uses `\n` for its newlines in the repository but I published it with `\r\n` when I cloned it on Windows because of my `auto.crlf` setting.

**Comment by rcjsuen** [comment]
@adelarsq This should be fixed in the new v0.0.12 release. Could you install it and give it a try?

**Comment by adelarsq** [comment]
@rcjsuen It's fixed. Thanks.

**Comment by Mdsp9070** [comment]
I have this same issue on Arch Linux with the latest version of docker-langserver

**Comment by Frederick888** [comment]
Can confirm. I fixed it locally for now. Perhaps set up CI/CD somewhere to publish new versions if @rcjsuen usually works under Windows?

**Comment by rcjsuen** [comment]
> I have this same issue on Arch Linux with the latest version of docker-langserver

@Mdsp9070 Apologies for not responding to this earlier. Looks like it got lost in my notifications. I'm surprised that this is happening since the `docker-langserver` file hasn't been touched in a while. I ran `npm install -g docker-langserver` on macOS and did not experience any problems with 0.3.0. 🤔

**Comment by Frederick888** [comment]
@rcjsuen It seems `npm` does its magic converting CRLF to LF. If you use `yarn global add dockerfile-language-server-nodejs` instead, the entry script will have CRLF under Linux/macOS, rendering it unusable.

**Comment by rcjsuen** [comment]
@Mdsp9070 @Frederick888 This should be fixed in the latest 0.4.1 release. Thank you again for the bug report.
